pages/receive/ReceiveSuccess.tsx

```diff
@@ -36,8 +36,8 @@ export function ReceiveSuccess() {
           </Text>
         </View>
         {getFiatAmount &&
-          <Text className="text-muted-foreground text-2xl font-bold">
-            {getFiatAmount(123) ?? ""}
+          <Text className="text-muted-foreground text-2xl font-semibold2">
```

Comment:
typo: `font-semibold`

Reply:
no, it's semibold2 and it's used across our codebase. for some reason I can't seem to manage to get the built-in tailwind classes to render other fonts :shrug:

Reply:
Oh didn't notice, sorry!